R/logging.R

```diff
@@ -187,8 +187,12 @@ tune_catalog <- function(issues) {
 
       # construct issue summary
       color <- if (current$type == "warning") {cli::col_yellow} else {cli::col_red}
+      pad <- nchar(pull(res[issue, "id"])) + 14L
```

Comment:
I would like a comment here describing where this 14 comes from, otherwise it turns into a magic number 😄